installation/tmpl/remove/default.php

```diff
@@ -96,15 +96,14 @@
 								<?php endforeach; ?>
 								</tbody>
 							</table>
-
 						<?php endif; ?>
 						<?php if ($this->development) : ?>
 							<div class="j-install-remove-check">
 								<div class="j-checkbox-group">
-									<label><input type="checkbox" checked> <?php echo Text::_('INSTL_SITE_DEVMODE_DESC'); ?></label>
+									<a href="javascript:" id="removeInstallationFolder"><?php echo Text::_('INSTL_COMPLETE_REMOVE_FOLDER'); ?></a>
```

Comment:
`href="javascript:"` ?

Reply:
I forgot to use a semicolon or `void(0)`. Thanks for this issue.

Reply:
@ahamed07sajeeb please confirm this has been handled or raise this as a separate issue, referencing this PR so as that charlie's effort is not lost